migrations/versions/a8fece80ca07_initial.py

```diff
@@ -0,0 +1,68 @@
+"""initial
+
+Revision ID: a8fece80ca07
+Revises: 
+Create Date: 2022-09-16 14:29:11.370606
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a8fece80ca07'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('ix_user_details_email', table_name='user_details')
```

Comment:
Here migration is partially created. Please drop the DB. Also delete this migration file. Then freshly generate a new only. That will have the complete migration script. 

If you face any difficulty on this, kindly let me know.